tests/Flatbuffers.Test.Swift.WASM/Sources/WASM/WASM.swift

```diff
@@ -0,0 +1 @@
+public struct WASM {}
```

Comment:
Same comment about capitalization as above.

```suggestion
public struct Wasm {}
```